mrt-services/docker-compose.yml

```diff
@@ -116,6 +116,23 @@ services:
     ports:
     - published: 3306
       target: 3306
+  ldap:
```

Comment:
When we add a component to the stack that can write to LDAP, it will be good to persist the content to a docker volume.  Otherwise, any data modifications made after the build will not persist on shutdown.

Reply:
Good idea.
We can also use a weekly export cron, much like we have on stage/prod.   We can then use the exported LDIF file as the input to the docker container startup.